README.md

```diff
@@ -13,18 +13,18 @@ from a Youtube Channel/Username or one-or-more Playlists.
 
 It downloads the video (`webm` or `mp4` extension – optionally
 recompress them in lower-quality, smaller size), the thumbnails, the
-subtitles and the authors' profile pictures ; then, it create a static
-HTML files folder of it before creating a ZIM off of it.
+subtitles and the authors' profile pictures ; then produces JSON files
+containing content for the channel, playlists, and videos, which are then utilized by a Vue.js UI.
 
```

Comment:
We need an additional statement explaining a bit the existence of the Vue.JS part. Something like "UI is a Vue.JS application which needs to be compiled as a static website with Vite and is them embedded inside the ZIM."

Reply:
Updated in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4).